application/src/main/java/org/togetherjava/tjbot/commands/Features.java

```diff
@@ -2,12 +2,16 @@
 
 import net.dv8tion.jda.api.JDA;
 import org.jetbrains.annotations.NotNull;
-import org.togetherjava.tjbot.commands.basic.*;
+import org.togetherjava.tjbot.commands.basic.PingCommand;
+import org.togetherjava.tjbot.commands.basic.RoleSelectCommand;
+import org.togetherjava.tjbot.commands.basic.SuggestionsUpDownVoter;
+import org.togetherjava.tjbot.commands.basic.VcActivityCommand;
```

Comment:
y u added separate imports?

Reply:
i guess the TJ-Bot Formatter(Google-Java StyleGuide) formatted it like this. Wildcard imports (.*) should be avoided. 

Can i close the conversation?